lib/module.js

```diff
@@ -447,6 +447,10 @@ Module._load = function(request, parent, isMain) {
   }
 
   var module = new Module(filename, parent);
+  if (opts !== undefined && opts.dlopenFlags) {
```

Comment:
note that 'object' type is checked below. I was expecting `Module._load` to be called strictly with either 'undefined' or 'object'. Being internal, I thought I could expect that. let me know if that sounds ok.